packages/react-reconciler/src/ReactFiberWorkLoop.new.js

```diff
@@ -3112,6 +3165,45 @@ export function captureCommitPhaseError(sourceFiber: Fiber, error: mixed) {
   }
 }
 
+export function captureCommitPhaseErrorForUnmountedFiber(
+  sourceFiber: Fiber,
+  nearestMountedAncestor: Fiber,
```

Comment:
Same thing here. You can just add this argument to `captureCommitPhaseError` and pass `fiber.return` in the normal case.

Reply:
Same as above. I could rejoin them, but the initial feedback I got was to separate them. I don't really feel strongly either way in this case.